featuretools/tests/synthesis/test_dfs_method.py

```diff
@@ -140,3 +167,43 @@ def test_dask_kwargs(entities, relationships):
     for column in feature_matrix:
         for x, y in zip(feature_matrix[column], feature_matrix_2[column]):
             assert ((pd.isnull(x) and pd.isnull(y)) or (x == y))
+
+
+def test_accepts_relative_training_window(datetime_es):
+    feature_matrix, features = dfs(entityset=datetime_es,
+                                   target_entity="transactions")
+
+    feature_matrix_2, features_2 = dfs(entityset=datetime_es,
+                                       target_entity="transactions",
+                                       cutoff_time=pd.Timestamp("2012-4-1 04:00"))
+
+    feature_matrix_3, features_3 = dfs(entityset=datetime_es,
+                                       target_entity="transactions",
+                                       cutoff_time=pd.Timestamp("2012-4-1 04:00"),
+                                       training_window=Timedelta("3 months"))
+
+    feature_matrix_4, features_4 = dfs(entityset=datetime_es,
+                                       target_entity="transactions",
+                                       cutoff_time=pd.Timestamp("2012-4-1 04:00"),
+                                       training_window="3 months")
+
+    # Test case for leap years
+    feature_matrix_5, features_5 = dfs(entityset=datetime_es,
+                                       target_entity="transactions",
+                                       cutoff_time=pd.Timestamp("2013-2-28 04:00"),
+                                       training_window=Timedelta("1 year"))
+
+    assert feature_matrix.index.all([1, 2, 3, 4, 5])
+    assert feature_matrix_2.index.all([1, 2, 3, 4])
+    assert feature_matrix_3.index.all([2, 3, 4])
+    assert feature_matrix_4.index.all([2, 3, 4])
+    assert feature_matrix_4.index.all([2, 3])
```

Comment:
I don't think these assertions are working as intended.  `assert feature_matrix_4.index.all([2, 3, 4])` and `assert feature_matrix_4.index.all([2, 3])` are both evaluating as True